### ISSUE TITLE
Subtitle files are not moved from drop folder

### ISSUE THREAD

**Issue by GoogleCodeExporter** [dizzydezz/jmm#359]
```
Subtitle files are not moved from drop folder (.ass and .srt files) when the 
video file is moved
```

Original issue reported on code.google.com by `werndly...@gmail.com` on 18 Dec 2012 at 3:12

**Comment by GoogleCodeExporter** [comment]
```
Also sub and idx files
```

Original comment by `werndly...@gmail.com` on 18 Dec 2012 at 3:50

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 3.1.26

ssa files as well
```

Original comment by `werndly...@gmail.com` on 18 Dec 2012 at 5:27
- Changed state: **Fixed**
